packages/react-async/src/tests/component.test.tsx

```diff
@@ -156,8 +158,9 @@ describe('createAsyncComponent()', () => {
       const renderKeepFresh = () => <KeepFresh />;
       const AsyncComponent = createAsyncComponent({load, renderKeepFresh});
 
-      // Enzyme doesn't know how to handle components that only return fragments.
       const keepFresh = mount(
+        // Enzyme doesn't know how to handle components that only return fragments.
+        // eslint-disable-next-line shopify/jsx-prefer-fragment-wrappers
```

Comment:
I'd just disable this rule for the whole file, I don't think it adds any value for tests.